File: veronymous-android-app/app/android-veronymous-client/src/main/java/io/veronymous/android/veronymous/client/tunnel/VeronymousTunnel.java
package io.veronymous.android.veronymous.client.tunnel;

import android.util.Log;

import com.wireguard.android.backend.Tunnel;

@Deprecated
public class VeronymousTunnel implements Tunnel {

    private static final String TAG = VeronymousTunnel.class.getSimpleName();

    private static final String TUNNEL_NAME = "veron0";

    @Override
    public String getName() {
        return TUNNEL_NAME;
    }

    @Override
    public void onStateChange(State newState) {
        Log.d(TAG, String.format("State change. NEW STATE %S", newState));
    }
}

File: veronymous-android-app/app/src/main/java/io/veronymous/android/vpn/app/wg/VeronymousTunnel.kt
package io.veronymous.android.vpn.app.wg

import android.util.Log
import com.wireguard.android.backend.Tunnel

class VeronymousTunnel : Tunnel {

    companion object {
        private val TAG = VeronymousTunnel::class.simpleName

        private const val NAME = "veron0";
    }

    override fun getName(): String {
        return NAME
    }

    override fun onStateChange(newState: Tunnel.State) {
        Log.d(TAG, "Got new state $newState")
        // TODO: Handle this
    }
}